packages/server-wallet/src/wallet/wallet-response.ts

```diff
@@ -207,6 +213,7 @@ export class WalletResponse {
     return {
       outbox: mergeOutgoing(this.outbox),
       channelResult: this.channelResults[0],
+      newObjective: this.createdObjectives[0],
       // objectivesToApprove: this.objectivesToApprove, // TODO: re-enable
```

Comment:
This line is probably unnecessary / outdated at this point and can be removed reasonably within the scope of this PR.

Reply:
Hmm yeah, I had no idea what to make of this comment.

It seems to as if we need to figure out how "objective approval" needs to work. But I'm happy to remove these TODOs -- they don't seem to add any value anymore.